File: server/makc2022--nodejs--nestjs/src/layer5/server/pages/dummy-main/item/queries/get/dummy-main-item-page-get-query-input.ts
/** Copyright (c) 2022 Maxim Kuzmin. All rights reserved. Licensed under the MIT License. */

import { DomainItemGetQueryInput as DummyMainDomainItemGetQueryInput } from 'src/layer4/domains/dummy-main/queries/item/get/domain-item-get-query-input';

/** Входные данные запроса на получение страницы сущности "DummyMain". */
export class DummyMainItemPageGetQueryInput {
  /** Входные данные запроса на получение элемента в домене "DummyMain". */
  inputOfDummyMainDomainItemGetQuery = new DummyMainDomainItemGetQueryInput();
}

File: server/makc2022--dotnet/Makc2022.Layer5.Sql.Server/Pages/DummyMain/Item/Queries/Get/DummyMainItemPageGetQueryInput.cs
// Copyright (c) 2022 Maxim Kuzmin. All rights reserved. Licensed under the MIT License.

using DummyMainDomainItemGetQueryInput = Makc2022.Layer4.Sql.Domains.DummyMain.Queries.Item.Get.DomainItemGetQueryInput;

namespace Makc2022.Layer5.Sql.Server.Pages.DummyMain.Item.Queries.Get
{
    /// <summary>
    /// Входные данные запроса на получение страницы сущности "DummyMain".
    /// </summary>
    public class DummyMainItemPageGetQueryInput
    {
        #region Properties

        /// <summary>
        /// Входные данные запроса на получение элемента в домене "DummyMain".
        /// </summary>
        public DummyMainDomainItemGetQueryInput InputOfDummyMainDomainItemGetQuery { get; } = new();

        #endregion Properties
    }
}
